src/map/Map.module.css

```diff
@@ -1,3 +1,8 @@
 .contextMenu {
     z-index: 3;
 }
+
+.mapContainer {
+    height: 100%;
+    width: 100%;
+}
```

Comment:
Not sure, but I needed this to make the map visible

Reply:
Yes, you're right this also works. I changed it.

Reply:
Hm, this works in Firefox, Chrome and also when using the mobile view in dev tools. But for the aws deployment the map stayed invisible on mobile for @karussell and me (even though it worked for desktop). So I reverted it to my initial version.

Reply:
Hm, this is odd.

Reply:
Yes, but I tried it twice and we both saw the same results. Feel free to try it again, but otherwise I'll leave as it is this for now.